src/components/teasers/grid/teaser-grid.tsx

```diff
@@ -25,13 +39,26 @@ const StyledTeaserGrid = styled.div<{ amountOfChildren: number }>`
   gap: 24px;
 
   justify-items: stretch;
+
+  /**
+   * we give the last element 24px more space for the padding 
+   * -> distance to outer window
+   */
   grid-template-columns: ${(props) =>
-    `repeat(${props.amountOfChildren}, minmax(224px, 1fr))`};
+    `repeat(${props.amountOfChildren - 1}, minmax(224px, 1fr)) 248px`};
+
+  > :last-child {
+    padding-right: 24px;
```

Comment:
This took me ages to figure out 😄 I was always thinking of an `::after` element like it's necessary when you do it with Flexbox (https://stackoverflow.com/questions/38993170/last-margin-padding-collapsing-in-flexbox-grid-layout)